x/dualstaking/keeper/delegate.go

```diff
@@ -561,3 +561,23 @@ func (k Keeper) getUnbondHoldBlocks(ctx sdk.Context, chainID string) uint64 {
 
 	// NOT REACHED
 }
+
+// GetDelegatorProviders gets all the providers the delegator is delegated to
+func (k Keeper) GetDelegatorProviders(ctx sdk.Context, delegator string) ([]string, error) {
+	var delegatorEntry types.Delegator
+	prefix := types.DelegatorKey(delegator)
+	nextEpoch, err := k.getNextEpoch(ctx)
```

Comment:
what is the reason to return the future value (of next epoch) and not the current value?
and if so, should we not clarify that some delegations will be in effect only in next epoch?
or better, add a flag to select between the state now and by next epoch.

Reply:
added `--show-pending` flag that lets you see pending delegators (applied in the next epoch). see https://github.com/lavanet/lava/commit/aaefdd332ea2388ff5aa28e3e320f79875219754